src/resources/views/crud/fields/upload_multiple.blade.php

```diff
@@ -40,7 +40,6 @@
 		<input
 	        type="file"
 	        name="{{ $field['name'] }}[]"
-	        value="@if (old(square_brackets_to_dots($field['name']))) old(square_brackets_to_dots($field['name'])) @elseif (isset($field['default'])) $field['default'] @endif"
```

Comment:
Pedro, I see the value is removed entirely here. Not even `default` registers any more. Was this intentional? Doesn't it have any downsides? Granted, I don't know who would set a `default` on this field... and why... and how it would work.

But I'm just wondering, since I see the `upload` field does have it - was this a mistake?

Reply:
No need to add anything under `value=` in `file` inputs. Don't have any effect..